backend/tools/add_slot_from_db_for_schedule_admin.py

```diff
@@ -0,0 +1,22 @@
+import db.models as models
+from db.models import Slots, SlotsShema
+
+
+def add_slot_from_db_for_schedule_admin(start, end):
+    """Add new free slots in db for admin schedule
+    """
+    try:
+        slots = models.Slots(start_interval=start, end_interval=end)
+        models.db.session.add(slots)
+        models.db.session.commit()
+        return get_last_slot_id()
```

Comment:
We have a single thread app so far, but in reality you would probably want to run multiple instances of the app (threads or processes) to process many requests simultaneously. 

In that case between commit() and get_last_slot_id() the database can change. In that case get_last_slot_id() will return an id of some other record.